docs/includes/shared-runtime-requirements-note.md

```diff
@@ -1,6 +1,9 @@
 > [!IMPORTANT]
-> Note that the shared JavaScript runtime is only available on the following platforms.
+> Note that the shared JavaScript runtime requirement set is only available on the following platforms.
+>
 > - Excel on the web, Windows, and Mac.
 > - PowerPoint on Windows (build 13218.10000 or later). The shared JavaScript runtime for PowerPoint is currently in preview and subject to change. It is not supported for use in production environments. To get the latest build you need to [join Office Insider](https://insider.office.com/join). A good way to try out preview features is by using a Microsoft 365 subscription. If you don't already have a Microsoft 365 subscription, you can get one by joining the [Microsoft 365 developer program](https://developer.microsoft.com/office/dev-program).
 >
 > At this time, the shared JavaScript runtime is not supported on iPad or in one-time purchase versions of Office 2019 or earlier.
+>
```

Comment:
Would be preferable to not bring in the event-based activation discussion into the note since the note shows up in articles that don't have anything to do with Outlook. Would be nice if we could keep it to a once sentence "Outlook doesn't use the SharedRuntime requirement set. For more info see...." approach.